pkg/kp/consulutil/watch.go

```diff
@@ -229,6 +229,8 @@ func WatchDiff(
 	outCh := make(chan *WatchedChanges)
 
 	go func() {
+		defer close(outCh)
```

Comment:
I think it was in its own commit, but I squashed it, so it's looks that way now.